docs/metrics/v3.md

```diff
@@ -32,51 +32,78 @@ These are metrics about requests served by the (current) node.
 |-----------------|--------------------------------------------------|
 | `/api/requests` | Metrics over all requests                        |
 | `/bucket/api`   | Metrics over all requests for a given bucket     |
-|                 |                                                  |
 
 ### Audit metrics
 
-These are metrics about the minio audit functionality
+These are metrics about the MinIO audit functionality.
 
 | Path     | Description                            |
 |----------|----------------------------------------|
 | `/audit` | Metrics related to audit functionality |
 
+### Cluster metrics
+
+These present metrics about the whole MinIO cluster.
+
+| Path                     | Description                 |
+|--------------------------|-----------------------------|
+| `/cluster/health`        | Cluster health metrics      |
+| `/cluster/usage/objects` | Object statistics           |
+| `/cluster/usage/buckets` | Object statistics by bucket |
+| `/cluster/erasure-set`   | Erasure set metrics         |
+
+### Debug metrics
+
+These are metrics for debugging
+
+| Path                        | Description                                       |
+|-----------------------------|---------------------------------------------------|
+| `/debug/go`                 | Standard Go lang metrics                          |
```

Comment:
```suggestion
| `/debug/go`                 | Go runtime metrics                          |
```

Reply:
Thanks! I'll pick this up after I'm done arguing with `csplit` 😄